libs/langchain-openai/src/tests/azure/chat_models.int.test.ts

```diff
@@ -0,0 +1,829 @@
+import { test, jest, expect } from "@jest/globals";
+import {
+  BaseMessage,
+  ChatMessage,
+  HumanMessage,
+  SystemMessage,
+} from "@langchain/core/messages";
+import { ChatGeneration, LLMResult } from "@langchain/core/outputs";
+import { ChatPromptValue } from "@langchain/core/prompt_values";
+import {
+  PromptTemplate,
+  ChatPromptTemplate,
+  HumanMessagePromptTemplate,
+  SystemMessagePromptTemplate,
+} from "@langchain/core/prompts";
+import { CallbackManager } from "@langchain/core/callbacks/manager";
+import { NewTokenIndices } from "@langchain/core/callbacks/base";
+import { InMemoryCache } from "@langchain/core/caches";
+import { getEnvironmentVariable } from "@langchain/core/utils/env";
+import {
+  ClientSecretCredential,
+  getBearerTokenProvider,
+} from "@azure/identity";
+import { AzureChatOpenAI } from "../../azure/chat_models.js";
+
+test("Test ChatOpenAI", async () => {
+  const chat = new AzureChatOpenAI({
+    modelName: "gpt-3.5-turbo",
+    maxTokens: 10,
+  });
+  const message = new HumanMessage("Hello!");
+  const res = await chat.call([message]);
+  console.log({ res });
```

Comment:
Ideally these also `expect` something, so the test doesn't pass by default unless an error is thrown. E.g expect tokens > 5 or something

Reply:
These tests are direct Azure equivalents of the ones in 1. [chat_models.int.test.ts](https://github.com/langchain-ai/langchainjs/blob/main/libs/langchain-openai/src/tests/chat_models.int.test.ts) 2. [embeddings.int.test.ts](https://github.com/langchain-ai/langchainjs/blob/main/libs/langchain-openai/src/tests/embeddings.int.test.ts) 3. [llms.int.test.ts](https://github.com/langchain-ai/langchainjs/blob/main/libs/langchain-openai/src/tests/llms.int.test.ts). Also, in other libs such as cloudflare, google-genai, etc. 

So, this follows the same pattern as these test cases.